src/Register-Login Page/Register.tsx

```diff
@@ -60,7 +60,7 @@ class Register extends React.Component<RegisterProps, RegisterState> {
             name="firstName"
             required={true}
             className="nameDiv"
-            onChange={this.handleFormChange}
+            onChange={e => this.handleFormChange(e)}
```

Comment:
Is there a reason why we adding the arrow functions back in for all of the event handlers? I don't think it is necessary.

Reply:
it seems to lose its bindings without the arrow functions, it comes back as undefined

Reply:
Ahh, I see why now, its because the class methods are defined using the normal function syntax. Okay, cool.